CHANGELOG.md

```diff
@@ -2,7 +2,14 @@
 
 Full documentation for RPP is available at [https://rocm.docs.amd.com/projects/rpp/en/latest](https://rocm.docs.amd.com/projects/rpp/en/latest)
 
-## (Unreleased) RPP 1.9.4
+## RPP 1.9.5 (Unreleased) 
```

Comment:
```suggestion
##  (Unreleased) RPP 1.9.5
```

The unreleased versions need to be labelled with (Unreleased) first in order to not be picked up by the release notes script.